id/documentation/installation/index.md

```diff
@@ -154,73 +200,69 @@ Informasi lebih lanjut terkait Ruby dan ekosistem sekelilingnya pada FreeBSD
 dapat ditemukan di [FreeBSD Ruby Project website][freebsd-ruby].
 
 
-### Ruby di Solaris dan OpenIndiana
-{: #solaris}
-
-Ruby 1.8.7 tersedia untuk Solaris 8 hingga Solaris 10 di
-[Sunfreeware][sunfreeware] dan Ruby 1.8.7 tersedia di
-Blastwave.
-Ruby 1.9.2p0 juga tersedia di [Sunfreeware][sunfreeware],
-namun sudah tidak diperbarui lagi.
+### Ruby pada OpenIndiana
+{: #openindiana}
 
 Untuk memasang Ruby pada [OpenIndiana][openindiana], mohon gunakan
-[Image Package Systems (IPS)][opensolaris-pkg] *client*.
+Image Package Systems (IPS) *client*.
 Ini akan memasang *binary* Ruby terbaru dan RubyGems secara langsung
-dari jaringan *repository* OpenSolaris untuk Ruby 1.9. Ini mudah:
+dari *repository* OpenSolaris. Ini mudah:
 
 {% highlight sh %}
-$ pkg install runtime/ruby-18
+$ pkg install runtime/ruby
 {% endhighlight %}
 
-Namun demikian, kakas bantu pihak ketiga mungkin adalah jalan terbaik untuk mendapatkan
-Ruby versi terbaru.
+Namun demikian, alat bantu pihak ketiga mungkin adalah jalan terbaik untuk
+mendapatkan versi Ruby terbaru.
 
 
 ### Distribusi Lainnya
 {: #other-systems}
 
 Pada sistem lain, Anda dapat mencari *package repository* dari
-*manager* distribusi Linux Anda untuk Ruby, atau kakas bantu pihak ketiga mungkin
-pilihan tepat untuk Anda.
+*manager* distribusi Linux Anda untuk Ruby. Kemungkinan lain, Anda dapat
+menggunakan sebuah [third-party installer][installers].
```

Comment:
I think it's better to use italic for this sentence.

`[*third-party installer*][installers]`

Reply:
OK. Just changed that sentence to italic, @kuntoaji.